src/entry-points/popup/VolumeIndicator.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+  import { tippyActionAsyncPreload as tippy } from "./tippyAction";
+  import { getMessage } from "@/helpers";
+  import { TelemetryMessage } from "../content/AllMediaElementsController";
+  import {
+    HotkeyAction_INCREASE_VOLUME,
+    HotkeyAction_DECREASE_VOLUME,
+  } from "@/hotkeys";
+
+  const tippyThemeMyTippyAndPreLine = "my-tippy white-space-pre-line";
+
+  export let getActionString: Function;
```

Comment:
I'd appreciate a better type.
Perhaps `import type { ... } from './App.svelte'` would be fine.